app/views/pseuds/delete_preview.html.erb

```diff
@@ -1,28 +1,27 @@
-<h2 class="heading"><%=h 'Pseud deletion' %></h2>
+<h2 class="heading"><%=h ts("Pseud deletion") %></h2>
 
 <p>
-  When you delete your pseud, any works, series, or comments you have created under it will be 
-  transferred to your default pseud.
+  <%= ts("When you delete your pseud, any works, series, or comments you have created under it will be 
+  transferred to your default pseud.") %>
 </p>
 
 <% if @pseud.bookmarks %>
   <%= form_tag({:action => "destroy"}, :method => :delete) do %>
-    <p>You have saved <%= @pseud.bookmarks.count %> bookmark(s) using this pseud. You can choose whether to
-    delete them or transfer them to your default pseud.
+    <p> <%= ts("You have saved  #{pluralize(@pseud.bookmarks.count, "bookmark")} using this pseud. You can choose whether to delete them or transfer them to your default pseud.") %>
```

Comment:
For [pluralization](https://guides.rubyonrails.org/i18n.html#pluralization), you'll need to add at least 2 translations, "one" and "other", for the default English language.

Reply:
Also, you shouldn't need to use `pluralize` directly. Once you set up the correct translation strings, you can simply use the `%{count}` placeholder in them.